.github/workflows/update-syscalls.yml

```diff
@@ -0,0 +1,42 @@
+name: Update Syscalls
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 2 1 * *'
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  update-syscalls:
+    runs-on: ubuntu-latest
+    container:
+      image: falcosecurity/syscalls-bumper:latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Git set-up
+        run: |
+          apt-get update && apt-get install -y --no-install-recommends git && apt-get clean
```

Comment:
`falcosecurity/syscalls-bumper:latest` is alpine based; we need:
`apk update && apk add git` ;)